### DISCUSSION TITLE
Is it possible to send data via usb to the gp2040-ce device.

### DISCUSSION THREAD

**Discussion by ryanMeredith** [OpenStickCommunity/GP2040-CE#1229] [Q&A] [answered]
Hello fantastic people.

I read through the documentation but couldn't see anything. 

Is it possible to send data from the parent device, a raspberry pi running a retopie stack, to a pico running gp2040 via usb?

I would love to turn on/off button lights depending on the console/game selected.

Many thanks

Ryan

**Comment by TheTrainGoes** [comment, marked as answer]
Hi @ryanMeredith ,

This is not possible though the current firmware.  Its recommended to use another device for LED control.
